internal/oci/containerd.go

```diff
@@ -25,6 +26,10 @@ import (
 	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
 )
 
+const (
+	backupDir = ".backup"
```

Comment:
Need to check if this creates an issue with how containerd reads mirror configuration.